README.md

```diff
@@ -19,11 +19,10 @@ MRTK-Unity is a Microsoft-driven project that provides a set of components and f
 
 If you're new to MRTK or Mixed Reality development in Unity, **we recommend you start at the beginning of our** [Unity development journey](https://docs.microsoft.com/windows/mixed-reality/unity-development-overview?tabs=mrtk%2Chl2) in the Microsoft Docs. The Unity development journey is specifically tailored to walk new developers through the installation, core concepts, and usage of MRTK. 
 
-> [!CAUTION]
-> The Unity development journey currently uses **MRTK version 2.4.0** and **Unity 2019.4**.
+| CAUTION: The Unity development journey currently uses **MRTK version 2.4.0** and **Unity 2019.4**. |
```

Comment:
@hferrone Can you change this from Caution to Important, like in the other branch?

Reply:
```suggestion
| IMPORTANT: The Unity development journey currently uses **MRTK version 2.4.0** and **Unity 2019.4**. |
```

FYI you can use the suggestion button @polar-kev  to adjust the sentence - then @hferrone just has to accept your change and there's no need for him to manually fix &push the change :)